src/javascript/detectors/stack_trace_exposure/stack_trace_exposure.js

```diff
@@ -27,7 +27,7 @@ var app = express()
 function stackTraceExposureCompliant() {
     app.get('www.example.com', (req, res) => {
         try {
-            var test = 10
+            throw new Error('')
```

Comment:
It's exactly the same as non compliant.

Reply:
No,  it is not the same example. As you can see in non-compliant case we  are passing `stackTrace` inside `res.send` but in compliant case we are just passing a string . This is the main difference between non-compliant and compliant case.